src/docs/user/ProActiveUserGuide.html

```diff
@@ -0,0 +1,3480 @@
+<!DOCTYPE html>
```

Comment:
Yes, this file is auto-generated from a ascidoc file. I will remove it, Thanks Mael